docker/Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN curl -sL \
       -o /usr/local/bin/opam && \
     chmod 755 /usr/local/bin/opam
 RUN opam init -y --comp=4.01.0 && \
+    opam switch 4.02.3 && \
```

Comment:
you can remove this line if you change the line above it to initialise the correct compiler:

```
  RUN opam init -y --comp=4.02.3 && \ 
```